File: Fornt-End/font-end-app/src/app/interfaces/actors/actor.ts
export interface actorDTO {
    id:number;
    name: string;
    birthdate: Date;
    photo: string;
    biography: string;
}

export interface actorCreationDTO {
    name: string;
    birthdate: Date;
    photo: File;
    biography: string;
}

export interface actorMovieDTO {
    id: number;
    name: string;
    character: string;
    photo: string;
}

File: Back-End/back-end-app/back-end-app/Entities/Actor.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace back_end_app.Entities
{
    public class Actor
    {
        public int Id { get; set; }
        [Required]
        [StringLength(maximumLength: 200)]
        public string Name { get; set; }
        public string Biography { get; set; }
        public DateTime Birthdate { get; set; }
        public string Photo {  get; set; }
        public List<MoviesActors> MoviesActors { get; set; }
    }
}
